phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/formats/hpo/HpoAnnotation.java

```diff
@@ -218,12 +239,16 @@ public Builder citations(List<String> cites) {
         this.citations=cites;
```

Comment:
Wrap inputs with Objects.requireNonNull() if they shouldn't be null. Java 11 has a nice Objects.requireNonNullElse() to replace nulls with a default value if that makes sense.